src/mame/heathkit/h89.cpp

```diff
@@ -188,6 +207,45 @@ void h89_state::h89_mem(address_map &map)
 	m_mem_view[1](0x1800, 0x1fff).rom().region("maincpu", 0x1800).unmapw();
 }
 
+void h89_state::map_fetch(address_map &map)
+{
+	map(0x0000, 0xffff).lr8(NAME([this](offs_t offset)
+	{
+		return m1_r(offset);
+	}));
+}
```

Comment:
This seems overly convoluted – why can’t use use `map(0x0000, 0xffff).r(FUNC(h89_state::m1_r));` in the same way that e.g. src/mame/osborne/osborne1.cpp does?